Orange/widgets/data/tests/test_owcontinuize.py

```diff
@@ -47,3 +47,17 @@ def test_one_column_equal_values(self):
         # Normalize.NormalizeBySD
         self.widget.continuous_treatment = 2
         self.widget.unconditional_commit()
+
+    def test_one_column_nan_values_normalize_sd(self):
+        """
+        No crash on a column with NaN values and with selected option
+        normalize by standard deviation (Not the same issue which is
+        tested above).
+        GH-2144
+        """
+        table = Table("iris")
+        table[:, 2] = np.NaN
+        self.send_signal("Data", table)
+        # Normalize.NormalizeBySD
+        self.widget.continuous_treatment = 2
+        self.widget.unconditional_commit()
```

Comment:
Add a test where a single value is set to `nan`. The expected behaviour is that this value is ignored, but the column is still normalized.